tasks/bes-install.yml

```diff
@@ -0,0 +1,19 @@
+- name: Download the hygieia/UI from the GitRepo
+    git:
+    repo: 'https://github.com/hygieia/UI.git'
+    dest: "{{ destdir }}"
```

Comment:
please have a default declaration of the variable.

Reply:
@asa1997 
Updated the requested changes



@sumodgeorge